src/app/containers/Brand/index.jsx

```diff
@@ -1,9 +1,9 @@
 import React, { useContext } from 'react';
 import Brand from '@bbc/psammead-brand';
-import { bool, node } from 'prop-types';
+import { bool, node, oneOfType, func, shape, any } from 'prop-types';
 import { ServiceContext } from '#contexts/ServiceContext';
 
-const BrandContainer = ({ skipLink, scriptLink, ...props }) => {
+const BrandContainer = ({ skipLink, scriptLink, brandRef, ...props }) => {
```

Comment:
```suggestion
const BrandContainer = ({ skipLink, scriptLink, focusRef, ...props }) => {
```
Not sure here but wondering if this should be `focusRef` to indicate what the ref is for: 'focusing on the brand' component?